### DISCUSSION TITLE
How to use local images with Image component

### DISCUSSION THREAD

**Discussion by creionhub** [shellscape/jsx-email#204] [Q&A]
Hi,

I'm trying to configure my project to use local assets during development following the example in the docs as shown below.

```ts
import { Img } from 'jsx-email';

const baseUrl = import.meta.env.DEV
  ? import.meta.resolve('../assets/')
  : 'https://assets.example.com/';

const Email = () => {
  return <Img src={`${baseUrl}cat.jpg`} alt="Cat" width="300" height="300" />;
};
```

However the images always return a 404 not found even though the URL is constructed. image URL generated looks like 

`http://localhost:55420/@fs/Users/path-to-image`.

Does anyone have any ideas how to resolve that?

**Comment by shellscape** [comment]
v2.0.6 supports using local static images. You can read more about that here: https://jsx.email/docs/core/cli#preview
